src/sentry/static/sentry/app/views/projectSettings/index.jsx

```diff
@@ -104,7 +104,10 @@ const ProjectSettings = createReactClass({
             >
               {t('Alerts')}
             </ListLink>
-            <ListLink to={`/${orgId}/${projectId}/settings/environments/`}>
+            <ListLink
+              to={`/${orgId}/${projectId}/settings/environments/`}
+              isActive={loc => path.indexOf(loc.pathname) === 0}
```

Comment:
would startsWith work here? its a bit more explicit

Reply:
Yes, but I'd rather keep the implementation the same as everything else in this page